src/taipy/core/taipy.py

```diff
@@ -577,6 +584,23 @@ def create_scenario(
     return _ScenarioManagerFactory._build_manager()._create(config, creation_date, name)
 
 
+def create_global_data_node(config: DataNodeConfig) -> DataNode:
+    """Create and return a new data node from a data node configuration with GLOBAL scope.
+
+    Parameters:
+        config (DataNodeConfig^): The data node configuration. The configuration must have GLOBAL scope.
+    Returns:
+        The new global data node.
+    Raises:
+        DataNodeConfigIsNotGlobal^: If the data node configuration does not have GLOBAL scope.
+    """
+    # Check if the data node config has GLOBAL scope
+    if config.scope is not Scope.GLOBAL:
+        raise DataNodeConfigIsNotGlobal(config.id)  # type: ignore
+
+    return _DataManagerFactory._build_manager()._create_and_set(config, None, None)
```

Comment:
Instead of using `_create_and_set` should we use `_bulk_get_or_create` method?
Because we don't want to create the data node twice if it already exists.

```suggestion
    return _DataManagerFactory._build_manager()._bulk_get_or_create([config])
```

Reply:
You are right, I will update this and add a unittest

Reply:
Actually @jrobinAV, I think since it is a global data node, the user can have 2 data nodes from the same configuration if he want.

We don't need to restrict this.

Reply:
But that's not the meaning of `GLOBAL` scope data node config. The meaning is that it is unique and shared by all scenarios. Two scenarios from 2 different cycles (or not) should share the same data node if it is `GLOBAL`.
Plus, that's not how it works when creating the data nodes from scenario creation. When creating the scenario with a `GLOBAL` data node, we look for the data node if it exists. If the data node is created before the scenario, the scenario will re-use it. Which one should we select if multiple `GLOBAL` data nodes from the same config exist?

I would keep the same behavior whatever we create the data node from `create_global_data_node` or from `create_scenario`.

Reply:
Right, that makes sense. I will push the fix shortly.